noxfile.py

```diff
@@ -157,3 +158,12 @@ def migration_test(session):
     # clearing the migration data
     os.remove("alembic.ini")
     shutil.rmtree("test_migration")
+
+    client = Client()
+
+    instance = client.instance("sqlalchemy-dialect-test")
+    database = instance.database("compliance-test")
+
+    database.update_ddl(["DROP TABLE account", "DROP TABLE alembic_version"]).result(
+        120
+    )
```

Comment:
It's not very easy to make alembic delete its own service table, so doing it with a common Spanner objects